src/kdc/main.c

```diff
@@ -65,6 +65,7 @@ static const char *pid_file = NULL;
 static int rkey_init_done = 0;
 static volatile int signal_received = 0;
 static volatile int sighup_received = 0;
+static int tcp_listen_backlog;
```

Comment:
I believe this could be put into struct socksetup.  There is no need to make it a global since it is only used within a call to loop_setup_network().